### ISSUE TITLE
Switches not accessible

### ISSUE THREAD

**Issue by acierpinski** [Dogfalo/materialize#2777]
Switch fields are not accessible by tab key.

**Comment by acburst** [comment]
In the latest release, this issue has been fixed for checkboxes which switches extend so they should be tabbable.

TODO: add an active state to switches

**Comment by acburst** [comment]
Added in efac791
